docs/usage.md

```diff
@@ -146,7 +142,7 @@ We highly recommend the use of Docker or Singularity containers for full pipelin
 We highly recommend the use of Docker container for full pipeline reproducibility.
 :::
 
-The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
+<!-- The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation). -->
```

Comment:
why did you comment this out? all nf-core pipelines come with these configs

Reply:
because I did not provide the configs in nf-core. May it is wrong, I thought I should have kept this only if providing configs for fastqrepair in nf-core, but maybe I did not get the point.